apps/judicial-system/web/src/routes/PublicProsecutor/Tables/CasesReviewed.tsx

```diff
@@ -40,6 +40,26 @@ const CasesReviewed: FC<Props> = ({ loading, cases }) => {
     ),
   }
 
+  const getVerdictViewTag = (row: CaseListEntry) => {
+    const today = new Date()
+    const deadline = new Date(row.indictmentVerdictAppealDeadline ?? '')
+    const variant = !row.indictmentVerdictViewedByAll
+      ? 'red'
+      : today > deadline
+      ? 'mint'
+      : 'blue'
+    const message = !row.indictmentVerdictViewedByAll
+      ? strings.tagVerdictUnviewed
+      : today > deadline
```

Comment:
I'd think that we'd need to do this server side, since dates are not to be trusted in the client. What do you think?

Reply:
Discussed offline, we're gonna let it slide this time as it's mostly for display purposes so shouldn't cause harm